### PR TITLE
[AUD-1439] Move profile-page store to common

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Moving profile page store to common in prep for native profile-page work

### CODE REVIEW

File: src/common/store/index.ts [hunk 1]
```diff
@@ -91,7 +92,8 @@ export const reducers = {
     exploreCollections: exploreCollectionsReducer,
     tokenDashboard: tokenDashboardSlice.reducer,
     track
-  })
+  }),
+  profile: profileReducer
```
Comment (rickyrombo) [inline src/common/store/index.ts:96]: Should this move to `pages`?

File: src/pages/profile-page/ProfilePageProvider.tsx [hunk 2]
```diff
@@ -47,17 +47,18 @@ import { resizeImage } from 'utils/imageProcessingUtil'
 import { getPathname, NOT_FOUND_PAGE, profilePage } from 'utils/route'
 import { parseUserRoute } from 'utils/route/userRouteParser'
 
-import { ProfilePageProps as DesktopProfilePageProps } from './components/desktop/ProfilePage'
-import { ProfilePageProps as MobileProfilePageProps } from './components/mobile/ProfilePage'
-import * as profileActions from './store/actions'
-import { feedActions } from './store/lineups/feed/actions'
-import { tracksActions } from './store/lineups/tracks/actions'
+import * as profileActions from '../../common/store/pages/profile/actions'
```
Comment (rickyrombo) [inline src/pages/profile-page/ProfilePageProvider.tsx:50]: Should these be non-relative paths?

**Review by rickyrombo — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1439-move-profile-store

File: src/pages/profile-page/ProfilePageProvider.tsx (hunk 2 above)
Comment (dylanjeffers) [reply src/pages/profile-page/ProfilePageProvider.tsx:50]: yah good call, ugh vscode!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1439-move-profile-store

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1439-move-profile-store

**Pull request merged**